roles/setup_pgd/tasks/setup_pgd.yml

```diff
@@ -106,3 +106,18 @@
   ansible.builtin.import_tasks: validate_setup_pgd.yml
   when:
     - validate_only|bool or use_validation|bool
+
+# Exclusive to pgd setup authentication
+- name: Start service oidentd, if not started
+  ansible.builtin.service:
+    name: oidentd
+    state: started
```

Comment:
Did you ever try updating the `pg_hba` file with `ident` auth and the `pg_ident` file to enable the ident authentication?

Reply:
It's working currently. Do not know if it'll break things by adding it to the 'pg_hba'